### ISSUE TITLE
Improve connection between Dryad and ORCID

### ISSUE THREAD

**Issue by ryscher** [datadryad/dryad-product-roadmap#2732]
When datasets are created in Dryad, author's ORCID profiles should be updated to include the datasets. 
- What is required for authors to supply the permission needed to do this?
- What is required for the technical connection?

**Comment by ahamelers** [comment]
https://support.orcid.org/hc/en-us/articles/360006973893-Trusted-organizations

**Comment by ahamelers** [comment]
https://info.orcid.org/documentation/api-tutorials/api-tutorial-add-and-update-data-on-an-orcid-record/

**Comment by ryscher** [comment]
NSF grant applications now allow (require?) researchers to import information from their ORCID profile. We can expect researchers to start improving the accuracy of their ORCID profile.

**Comment by ahamelers** [comment]
Looking at what ORCID allows "trusted organizations" to do, I don't think there's anything more we want. Have we verified that the DataCite pipeline is definitely working?

**Comment by ryscher** [comment]
Well, it worked for me :) My Dryad dataset was published last yesterday, and a few hours later I received a notice from ORCID about a new entry to my profile. We could spot check other recent data publications....

**Comment by ahamelers** [comment]
Reports that Dryad data is not being consistently (or, majorly) added to ORCID profiles by DataCite. We may want to go back to the integration idea and at least give users the option of adding records through us

**Comment by rcurty** [comment]
It seems DataCite is not updating user ORCID profiles as expected. I couldn’t find examples at UCSB by searching for people with public profiles who have published in Dryad multiple times. Additionally, I couldn’t locate a dataset I published years ago on my own profile. I was able to enter it manually, but I’m wondering if there’s a setting or feature that needs to be enabled to pull datasets from Dryad automatically. I have Dryad listed as a trusted organization on my profile settings, however, access type is currently limited to "Read your information with visibility set to Trusted Parties".

**Comment by rcurty** [comment]
Looking at the thread, I wonder if DataCite should be listed as a trusted organization (it is not listed in my settings). If so, should that be accessed via the ORCID API for the whole campus?

**Comment by ryscher** [comment]
Yes, DataCite needs to be listed as a trusted organization. Dryad does not add the content directly, all of the content goes through DataCite. See instructions for [enabling the DataCite/ORCID connection](https://support.datacite.org/docs/datacite-and-orcid).

**Comment by ahamelers** [comment]
If users need to do an additional step to get their data listed on ORCID we should make that really clear and prompt them to do it!

**Comment by ryscher** [comment]
Some more info from ORCID:

Note that writing data to ORCID records is one of the most helpful things an organization can do to [contribute trustworthy research information](https://info.orcid.org/interpreting-the-trustworthiness-of-an-orcid-record/) back to the wider research and scholarly communication ecosystem, and prepare researchers for upcoming potential funder requirements from US federal agencies, due to NSPM-33, for example - [read about ORCID & NSPM-33](https://lyrasisnow.org/nspm-33-orcid-information-for-research-organizations/).
In some cases, your system integration may not be configured to write data to ORCID records. Check our [Common Systems page](https://orcidus.lyrasis.org/common-systems-for-integration/) to see the default ORCID API integration status for local software systems used at your organization. Consider using the write functionality if your integration allows it, or consider a custom ORCID API integration to get more value from the ORCID member API. See our [Custom Integration Guide](https://orcidus.lyrasis.org/technical-integration-guide/) for more details.
If you’re not already, consider using the [ORCID Affiliation Manager](https://info.orcid.org/affiliation-manager/) to write affiliation data to your researchers’ ORCID records via a CSV upload/email notification workflow.
